packages/parser-css/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
+    "postcss": "^7.0.1"
+  },
+  "description": "webhint parser needed to analyze CSS files",
+  "devDependencies": {
+    "ava": "^0.25.0",
+    "cpx": "^1.5.0",
+    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-markdown": "^1.0.0-beta.7",
+    "eslint-plugin-typescript": "^0.12.0",
+    "hint": "^3.0.0-beta.4",
+    "markdownlint-cli": "^0.11.0",
+    "npm-link-check": "^2.0.0",
+    "npm-run-all": "^4.1.2",
+    "nyc": "^12.0.2",
+    "proxyquire": "2.0.0",
+    "rimraf": "^2.6.2",
+    "sinon": "^6.1.4",
+    "typescript": "^2.9.2",
+    "typescript-eslint-parser": "^16.0.1"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "dist/src",
+    "npm-shrinkwrap.json"
+  ],
+  "homepage": "https://webhint.io/",
+  "keywords": [
+    "webhint",
+    "webhint-parser",
+    "css"
+  ],
+  "license": "Apache-2.0",
+  "main": "dist/src/parser.js",
+  "name": "@hint/parser-css",
+  "nyc": {
+    "extends": "../../.nycrc"
+  },
+  "peerDependencies": {
+    "hint": "^3.0.0-beta.4"
+  },
+  "repository": "webhintio/hint",
+  "scripts": {
+    "build": "npm run clean && npm-run-all build:*",
+    "build-release": "npm run clean && npm run build:assets && tsc --inlineSourceMap false --removeComments true",
+    "build:assets": "cpx \"./{src,tests}/**/{!(*.ts),.!(ts)}\" dist",
+    "build:ts": "tsc",
+    "clean": "rimraf dist",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint . --cache --ext js --ext md --ext ts --ignore-path ../../.eslintignore --report-unused-disable-directives",
+    "lint:md": "markdownlint --ignore CHANGELOG.md *.md",
+    "test": "npm run lint && npm run build && npm run test-only",
+    "test-only": "nyc ava",
+    "watch": "npm run build && npm-run-all --parallel -c watch:*",
+    "watch:assets": "npm run build:assets -- -w --no-initial",
+    "watch:test": "ava --watch",
+    "watch:ts": "npm run build:ts -- --watch"
+  },
+  "version": "1.0.0-beta.0"
```

Comment:
I think you need a `private: true`. Not sure about the version. @alrra?
Did you use `npm create parser` to do the scaffolding?

Reply:
@molant Thanks. I didn't use `npm create parser`, is that documented somewhere? Also, is that the correct command? It didn't do anything when I just tried to run it. Instead I used the current parser-javascript as a base and built out from there.

Also, is parser-webpack-config the most recently created parser? It seems to be the only one with `private: true` and a different version (`0.0.1`). The rest of the existing parsers seem to match what I did here. Happy to make the changes, but wanted to double-check due to the inconsistencies.

Reply:
> I think you need a private: true. Not sure about the version. @alrra?

If we don't want a beta version published, `private: true` and `v1.0.0`.

Reply:
@antross this is what I get when I run the command:

![image](https://user-images.githubusercontent.com/606594/43534624-aee0e81e-956c-11e8-83d8-23bd0c3349e0.png)

Maybe you have an old version of npm? Try with `npm init parser` just in case.

@alrra doesn't your script need to have the `private:true` initially to know it is going to be the first time it publishes something and create the changelog?

Reply:
> doesn't your script need to have the private:true initially to know it is going to be the first time it publishes something and create the changelog?

Yes, but more importantly the first tag, and not search for previous tags.

Reply:
> npm init parser just in case.

or `npm create parser`. :D

Reply:
>  I didn't use `npm create parser`, is that documented somewhere? 

`create` should just be an alias of `init` (I'm using `npm` `v6.2.0`):

```
$ npm init -h

npm init [--force|-f|--yes|-y|--scope]
npm init <@scope> (same as `npx <@scope>/create`)
npm init [<@scope>/]<name> (same as `npx [<@scope>/]create-<name>`)

aliases: create, innit  <===========
```

Reply:
Yep, I needed to upgrade my npm. That said, running through the wizard only gave me a `.npmrc` and `LICENSE.text` file in the created folder... seems like it's not working right...

Apart from that, I think we should document this command here: https://webhint.io/docs/contributor-guide/how-to/parser/. Happy to spin up a PR for that change if you agree.

Reply:
>  seems like it's not working right..

Yes, it seems to be a bug:

```
Files copied
(node:5576) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/Users/xxxxx/.npm/_npx/5572/lib/node_modules/create-parser/dist/src/templates/new-parser/script.hbs'
(node:5576) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:5576) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

> Apart from that, I think we should document this command here: https://webhint.io/docs/contributor-guide/how-to/parser/. Happy to spin up a PR for that change if you agree.

Do it!

Reply:
> Apart from that, I think we should document this command here: 

That site is not updated. The latest version of the documentation is in the staging environment until we have the final homepage design. The documentation for the parser is in https://sonarwhal-staging.azurewebsites.net/docs/contributor-guide/how-to/parser/

Reply:
> Yes, there seems to be a bug:

I fixed that bug a while ago but there's another one with one of the templates not being properly formed. I'm working on a fix.

Reply:
And the [fix is in master](https://github.com/webhintio/hint/commit/887ce5c086c1a0741e5702f4d7caf556aeb51fdb)